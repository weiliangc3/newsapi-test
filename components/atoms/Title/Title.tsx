import styled from 'styled-components';

export default styled.h1`
  font-size: 32px;
  text-align: center;
  font-family: ${(props) => props.theme.headerFont};
  color: ${(props) => props.theme.mainColour};
`;
