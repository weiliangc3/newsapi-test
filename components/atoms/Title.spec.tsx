import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Title from './Title';

// test written as starter
it('renders correctly', () => {
  const tree = render(<Title>A Title</Title>);
  expect(tree.container).toMatchSnapshot();
});
