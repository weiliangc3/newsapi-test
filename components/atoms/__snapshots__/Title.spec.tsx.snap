// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    class="sc-bdvvtL jQFIVW"
  >
    A Title
  </h1>
</div>
`;
